tests/performanceplatform/collector/piwik/test_realtime.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import json
+from mock import patch
+from hamcrest import assert_that, equal_to
+from freezegun import freeze_time
+from performanceplatform.collector.piwik.realtime import main, Fetcher, Parser
+from tests.performanceplatform.collector.piwik.test_query import(
+    query, credentials)
+
+
+def get_piwik_data():
+    with open("tests/fixtures/piwik_realtime.json", "r") as f:
+        return json.loads(f.read())
+
+
+def build_fetcher(query_params={}):
```

Comment:
these methods could be in a utils file and imported into both tests to avoid duplication